CRM/Financial/BAO/FinancialTypeAccount.php

```diff
@@ -75,59 +75,65 @@ public static function add(&$params, $ids = NULL) {
    * Delete financial Types.
    *
    * @param int $financialTypeAccountId
-   * @param int $accountId
+   * @param int $accountId  (not used)
    *
    * @throws \CRM_Core_Exception
+   * @deprecated
    */
   public static function del($financialTypeAccountId, $accountId = NULL) {
-    // check if financial type is present
-    $check = FALSE;
-    $relationValues = CRM_Core_PseudoConstant::get('CRM_Financial_DAO_EntityFinancialAccount', 'account_relationship');
+    static::deleteRecord(['id' => $financialTypeAccountId]);
```

Comment:
This BAO has a nonstandard naming convention. I think this will fix the test:

```suggestion
    CRM_Financial_DAO_EntityFinancialAccount::deleteRecord(['id' => $financialTypeAccountId]);
```

Reply:
Thanks @colemanw